src/PlusDataCollection/MicrosoftMediaFoundation/vtkPlusMmfVideoSource.cxx

```diff
@@ -614,6 +614,26 @@ PlusStatus vtkPlusMmfVideoSource::AddFrame(unsigned char* bufferData, DWORD buff
   {
     return PLUS_FAIL;
   }
+
+  const double maximumFrameTimeVariance = 0.2;  // to make sure we don't drop frames because of slight variance in acquisition rate, we allow up to 20% higher frame rate before we start dropping frames

```

Comment:
This will not work as `this->AcquisitionRate` is passed in `InternalConnect` as the desired webcam framerate.

You will need to add another variable in the XML to specify the throttled framerate.

Reply:
Or repurpose `this->AcquisitionRate` to mean throttled framerate and add another XML attribute like `WebcamFramerate` and update the docs/sample configs.

Reply:
MMF imaging devices support a set of frame rates predefined at the driver level and you may ask for 5fps but the closest you may get is 30fps.

this->AcquisitionRate means the desired (throttled, if necessary) frame rate.

Reply:
If you ask for 5fps and the device doesn't support it, the request will fail. The only exception is I added double rounding so if you ask for 60fps and the device supports 59.99999999 it will allow it.

Reply:
The MediaFoundationLibrary does not return closest match. We can change it to do that if we want to.

Reply:
Something isn't right, some of my edits are not in the MfCapture lib. Investigating.